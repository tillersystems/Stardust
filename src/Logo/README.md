# Logo

### Usage

```jsx
import { Logo } from '@tillersystems/stardust';
```

<!-- STORY -->

### Properties

- `color` - Logo color.
- `height` - Logo height.
- `width` - Logo width.

| propName | propType | defaultValue | isRequired |
| -------- | :------: | :----------: | :--------: |
| `color`  | `string` |   `white`    |     -      |
| `height` | `string` |     `28`     |     -      |
| `width`  | `string` |    `120`     |     -      |

### Example

```jsx
import { Logo } from '@tillersystems/stardust';

render() {
  return <Logo color="red" width="230" height="76" />
}
```
