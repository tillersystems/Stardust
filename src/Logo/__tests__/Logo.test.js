import React from 'react';
import 'jest-styled-components';

import Logo from '..';

describe('<Logo />', () => {
  test('should render without a problem', () => {
    const { container } = render(<Logo />);
    expect(container.firstChild).toMatchSnapshot();
  });

  test('should render with custom width & height', () => {
    const { getByTestId } = render(<Logo width="300" height="80" />);
    const logoNode = getByTestId('logo');
    expect(logoNode).toHaveAttribute('width', '300');
    expect(logoNode).toHaveAttribute('height', '80');
  });

  test('should render with custom color', () => {
    const color = 'hsl(217, 89%, 61%)';
    const { getByTestId } = render(<Logo color={color} />);
    const logoPathNode = getByTestId('logoPath');
    expect(logoPathNode).toHaveAttribute('fill', color);
  });
});
