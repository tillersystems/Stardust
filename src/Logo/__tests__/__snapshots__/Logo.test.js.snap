// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render with custom color 1`] = `
<Logo
  color="hsl(217, 89%, 61%)"
  height="59"
  theme={
    Object {
      "breakpoints": Object {
        "lg": 992,
        "md": 768,
        "sm": 576,
        "xl": 1200,
        "xs": 0,
      },
      "dimensions": Object {
        "big": "3.1rem",
        "bigInt": 3.1,
        "huge": "4.5rem",
        "hugeInt": 4.5,
        "medium": "1.5rem",
        "mediumInt": 1.5,
        "radius": "0.4rem",
        "radiusInt": 0.4,
        "small": "0.7rem",
        "smallInt": 0.7,
        "tiny": "0.3rem",
        "tinyInt": 0.3,
      },
      "fonts": Object {
        "family": "\\"Open Sans\\", Helvetica, Arial, sans-serif",
        "size": Object {
          "big": "1.6rem",
          "bigInt": 1.6,
          "default": "1.2rem",
          "defaultInt": 1.2,
          "h1": "3.6rem",
          "h1Int": 3.6,
          "h2": "3rem",
          "h2Int": 3,
          "h3": "2.6rem",
          "h3Int": 2.6,
          "h4": "2.2rem",
          "h4Int": 2.2,
          "h5": "2.0rem",
          "h5Int": 2,
          "h6": "1.8rem",
          "h6Int": 1.8,
          "huge": "6.4rem",
          "hugeInt": 6.4,
          "medium": "1.4rem",
          "mediumInt": 1.4,
          "small": "0.7rem",
          "smallInt": 0.7,
          "tiny": "0.3rem",
          "tinyInt": 0.3,
        },
        "spacing": Object {
          "big": "1.5px",
          "medium": "1px",
          "small": "0.5px",
        },
        "weight": Object {
          "normal": 400,
          "thick": 600,
          "thin": 200,
        },
      },
      "palette": Object {
        "avatar": Object {
          "alizarin": "hsl(6, 78%, 57%)",
          "carrot": "hsl(28, 80%, 52%)",
          "emerald": "hsl(145, 63%, 49%)",
          "river": "hsl(204, 70%, 53%)",
          "turquoise": "hsl(168, 76%, 42%)",
          "wisteria": "hsl(282, 44%, 47%)",
        },
        "black": "hsl(0, 0%, 0%)",
        "bodyBackground": "hsl(218, 29%, 95%)",
        "darkBlue": "hsl(213, 17%, 20%)",
        "darkGrey": "hsl(206,16%,60%)",
        "failure": Object {
          "dark": "#e74432",
          "darker": "#e42f1b",
          "default": "hsl(6, 79%, 65%)",
        },
        "googleBrandRed": "hsl(5, 81%, 56%)",
        "lightGrey": "hsl(207,22%,90%)",
        "mediumGrey": "hsl(206,23%,69%)",
        "mysticGrey": "hsl(216,45%,98%)",
        "primary": Object {
          "dark": "#1873a0",
          "darker": "#14638a",
          "default": "hsl(200, 74%, 46%)",
        },
        "secondary": Object {
          "default": "hsl(0, 100%, 100%)",
        },
        "spaceGrey": "hsl(207,13%,45%)",
        "success": Object {
          "dark": "#6da333",
          "darker": "#60902d",
          "default": "hsl(89, 53%, 52%)",
        },
        "warning": Object {
          "dark": "#f59a24",
          "darker": "#f48f0b",
          "default": "hsl(34, 91%, 65%)",
        },
        "white": "hsl(0, 100%, 100%)",
        "whiteOpacity": [Function],
      },
    }
  }
  width="180"
>
  <svg
    height="59"
    viewBox="0 0 1689 720"
    width="180"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M359.247 496.87c75.28 0 136.3-61.28 136.3-136.874 0-75.59-61.02-136.87-136.3-136.87-75.276 0-136.3 61.28-136.3 136.87 0 75.594 61.024 136.874 136.3 136.874zm174.341-141.8c2.701 0 4.91 2.217 4.91 4.926 0 2.714-2.209 4.932-4.91 4.932h-23.815c-1.215 38.053-16.407 72.541-40.61 98.473l16.833 16.904c1.91 1.915 1.91 5.055 0 6.973a4.915 4.915 0 0 1-6.944 0l-16.83-16.907c-25.825 24.309-60.171 39.561-98.063 40.787v23.91c0 2.712-2.211 4.932-4.912 4.932-2.7 0-4.912-2.22-4.912-4.932v-23.91c-37.891-1.226-72.235-16.478-98.06-40.787l-16.832 16.907a4.915 4.915 0 0 1-6.944 0c-1.907-1.918-1.907-5.058 0-6.973l16.833-16.904c-24.2-25.932-39.393-60.42-40.61-98.473h-23.813c-2.7 0-4.909-2.218-4.909-4.932 0-2.71 2.208-4.927 4.909-4.927h23.813c1.217-38.053 16.41-72.538 40.61-98.476l-16.833-16.901c-1.907-1.918-1.907-5.055 0-6.97a4.904 4.904 0 0 1 6.944 0l16.833 16.9c25.824-24.305 60.168-39.56 98.06-40.78v-23.916c0-2.709 2.21-4.926 4.911-4.926 2.7 0 4.912 2.217 4.912 4.926v23.916c37.892 1.22 72.238 16.475 98.062 40.78l16.83-16.9a4.904 4.904 0 0 1 6.945 0c1.91 1.915 1.91 5.052 0 6.97l-16.834 16.9c24.204 25.939 39.396 60.424 40.61 98.477h23.816zM359.25 247.77c61.624 0 111.76 50.343 111.76 112.229 0 1.752-.051 3.49-.135 5.223a5.944 5.944 0 0 1-2.134 4.306 5.848 5.848 0 0 1-4.598 1.321 12.685 12.685 0 0 0-12.572 5.96c-3.67 6-1.8 13.88 4.172 17.566a12.55 12.55 0 0 0 2.823 1.28 5.942 5.942 0 0 1 3.592 3.182 5.991 5.991 0 0 1 .078 4.819c-3.056 7.254-6.95 14.275-11.578 20.877a5.943 5.943 0 0 1-4.085 2.481 5.932 5.932 0 0 1-4.578-1.345c-.48-.399-.979-.789-1.522-1.127a12.605 12.605 0 0 0-9.611-1.54 12.614 12.614 0 0 0-7.878 5.732 12.792 12.792 0 0 0 1.05 14.83 5.964 5.964 0 0 1 1.31 4.615 5.972 5.972 0 0 1-2.498 4.09 112.016 112.016 0 0 1-19.324 10.79 5.904 5.904 0 0 1-4.832-.056 5.948 5.948 0 0 1-3.19-3.64 12.737 12.737 0 0 0-5.46-7.067 12.612 12.612 0 0 0-9.61-1.54 12.631 12.631 0 0 0-7.88 5.732l-.003.003a12.806 12.806 0 0 0-1.713 8.684 6 6 0 0 1-1.247 4.684 5.959 5.959 0 0 1-4.303 2.203c-1.997.101-4.008.164-6.034.164-61.627 0-111.763-50.342-111.763-112.227 0-61.886 50.136-112.228 111.763-112.228zm84.81 165.202c1.785-2.868 3.42-5.81 4.891-8.807-11.089-7.273-14.47-22.193-7.496-33.595a24.464 24.464 0 0 1 4.96-5.792h-28.033c-2.312 29.079-25.397 52.26-54.358 54.58v28.614a24.363 24.363 0 0 1 13.65-8.81c6.385-1.531 12.99-.476 18.581 2.979a24.56 24.56 0 0 1 7.714 7.557 99.744 99.744 0 0 0 7.427-4.156 24.742 24.742 0 0 1 1.475-23.05c3.431-5.615 8.847-9.556 15.233-11.084a24.246 24.246 0 0 1 15.956 1.564zm-184.58-48.194c1.128 24.057 10.731 45.923 25.878 62.647l28.713-28.83a59.431 59.431 0 0 1-13.954-33.808h-37.67c-.012 0-.021-.009-.033-.009h-2.934zm25.887-72.22c-15.15 16.72-24.756 38.59-25.887 62.646h40.635a59.442 59.442 0 0 1 13.938-33.82l-26.624-26.738c-.012-.009-.015-.03-.027-.042l-2.035-2.047zm69.12-29.658v-3.09c-23.95 1.137-45.725 10.774-62.379 25.981l28.692 28.812c9.227-7.887 20.877-12.984 33.681-14.006v-37.664c0-.012.006-.024.006-.033zm43.216 51.712l28.685-28.809c-16.648-15.207-38.414-24.853-62.364-25.992v40.786c12.805 1.025 24.455 6.125 33.679 14.015zm-38.453 90.356c24.74 0 44.803-20.143 44.803-44.99 0-24.848-20.062-44.988-44.803-44.988-24.747 0-44.806 20.14-44.806 44.988 0 24.847 20.059 44.99 44.806 44.99zm-38.435.396l-28.713 28.833c16.651 15.21 38.432 24.847 62.385 25.983v-40.82c-12.798-1.021-24.448-6.112-33.672-13.996zm97.57-50.16h40.62c-1.138-24.05-10.731-45.91-25.873-62.629l-28.688 28.81a59.445 59.445 0 0 1 13.941 33.819zM1108.2 313.19v94.305h74.274v13.329h-87.514V313.19h13.24zm-153.805 0v94.305h74.292v13.329h-87.532V313.19h13.24zm-318.875 0h98.695v13.325h-42.705v94.309h-13.27v-94.309h-42.72V313.19zm786.95 60.882l-.29.003v46.75h-13.273V313.19h67.405v-.045c16.597 0 30.052 13.464 30.052 30.152 0 16.649-13.455 30.166-30.052 30.166h-3.853l35.955 47.361h-17.072l-35.564-47.36-33.308.608zm-.29-47.556v33.82l54.132-.186c9.787 0 17.62-8.23 16.758-18.27-.758-8.829-8.6-15.364-17.424-15.364h-53.465zm-595.372 94.308V313.19h13.276v107.634h-13.276zm421.938-107.634h87.526v13.325h-74.247v33.82h47.289v13.317h-47.289v33.843h74.247v13.329h-87.526V313.19z"
      fill="hsl(217, 89%, 61%)"
      fillRule="evenodd"
    />
  </svg>
</Logo>
`;

exports[`<Logo /> should render with custom width & height 1`] = `
<Logo
  color="white"
  height="80"
  theme={
    Object {
      "breakpoints": Object {
        "lg": 992,
        "md": 768,
        "sm": 576,
        "xl": 1200,
        "xs": 0,
      },
      "dimensions": Object {
        "big": "3.1rem",
        "bigInt": 3.1,
        "huge": "4.5rem",
        "hugeInt": 4.5,
        "medium": "1.5rem",
        "mediumInt": 1.5,
        "radius": "0.4rem",
        "radiusInt": 0.4,
        "small": "0.7rem",
        "smallInt": 0.7,
        "tiny": "0.3rem",
        "tinyInt": 0.3,
      },
      "fonts": Object {
        "family": "\\"Open Sans\\", Helvetica, Arial, sans-serif",
        "size": Object {
          "big": "1.6rem",
          "bigInt": 1.6,
          "default": "1.2rem",
          "defaultInt": 1.2,
          "h1": "3.6rem",
          "h1Int": 3.6,
          "h2": "3rem",
          "h2Int": 3,
          "h3": "2.6rem",
          "h3Int": 2.6,
          "h4": "2.2rem",
          "h4Int": 2.2,
          "h5": "2.0rem",
          "h5Int": 2,
          "h6": "1.8rem",
          "h6Int": 1.8,
          "huge": "6.4rem",
          "hugeInt": 6.4,
          "medium": "1.4rem",
          "mediumInt": 1.4,
          "small": "0.7rem",
          "smallInt": 0.7,
          "tiny": "0.3rem",
          "tinyInt": 0.3,
        },
        "spacing": Object {
          "big": "1.5px",
          "medium": "1px",
          "small": "0.5px",
        },
        "weight": Object {
          "normal": 400,
          "thick": 600,
          "thin": 200,
        },
      },
      "palette": Object {
        "avatar": Object {
          "alizarin": "hsl(6, 78%, 57%)",
          "carrot": "hsl(28, 80%, 52%)",
          "emerald": "hsl(145, 63%, 49%)",
          "river": "hsl(204, 70%, 53%)",
          "turquoise": "hsl(168, 76%, 42%)",
          "wisteria": "hsl(282, 44%, 47%)",
        },
        "black": "hsl(0, 0%, 0%)",
        "bodyBackground": "hsl(218, 29%, 95%)",
        "darkBlue": "hsl(213, 17%, 20%)",
        "darkGrey": "hsl(206,16%,60%)",
        "failure": Object {
          "dark": "#e74432",
          "darker": "#e42f1b",
          "default": "hsl(6, 79%, 65%)",
        },
        "googleBrandRed": "hsl(5, 81%, 56%)",
        "lightGrey": "hsl(207,22%,90%)",
        "mediumGrey": "hsl(206,23%,69%)",
        "mysticGrey": "hsl(216,45%,98%)",
        "primary": Object {
          "dark": "#1873a0",
          "darker": "#14638a",
          "default": "hsl(200, 74%, 46%)",
        },
        "secondary": Object {
          "default": "hsl(0, 100%, 100%)",
        },
        "spaceGrey": "hsl(207,13%,45%)",
        "success": Object {
          "dark": "#6da333",
          "darker": "#60902d",
          "default": "hsl(89, 53%, 52%)",
        },
        "warning": Object {
          "dark": "#f59a24",
          "darker": "#f48f0b",
          "default": "hsl(34, 91%, 65%)",
        },
        "white": "hsl(0, 100%, 100%)",
        "whiteOpacity": [Function],
      },
    }
  }
  width="300"
>
  <svg
    height="80"
    viewBox="0 0 1689 720"
    width="300"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M359.247 496.87c75.28 0 136.3-61.28 136.3-136.874 0-75.59-61.02-136.87-136.3-136.87-75.276 0-136.3 61.28-136.3 136.87 0 75.594 61.024 136.874 136.3 136.874zm174.341-141.8c2.701 0 4.91 2.217 4.91 4.926 0 2.714-2.209 4.932-4.91 4.932h-23.815c-1.215 38.053-16.407 72.541-40.61 98.473l16.833 16.904c1.91 1.915 1.91 5.055 0 6.973a4.915 4.915 0 0 1-6.944 0l-16.83-16.907c-25.825 24.309-60.171 39.561-98.063 40.787v23.91c0 2.712-2.211 4.932-4.912 4.932-2.7 0-4.912-2.22-4.912-4.932v-23.91c-37.891-1.226-72.235-16.478-98.06-40.787l-16.832 16.907a4.915 4.915 0 0 1-6.944 0c-1.907-1.918-1.907-5.058 0-6.973l16.833-16.904c-24.2-25.932-39.393-60.42-40.61-98.473h-23.813c-2.7 0-4.909-2.218-4.909-4.932 0-2.71 2.208-4.927 4.909-4.927h23.813c1.217-38.053 16.41-72.538 40.61-98.476l-16.833-16.901c-1.907-1.918-1.907-5.055 0-6.97a4.904 4.904 0 0 1 6.944 0l16.833 16.9c25.824-24.305 60.168-39.56 98.06-40.78v-23.916c0-2.709 2.21-4.926 4.911-4.926 2.7 0 4.912 2.217 4.912 4.926v23.916c37.892 1.22 72.238 16.475 98.062 40.78l16.83-16.9a4.904 4.904 0 0 1 6.945 0c1.91 1.915 1.91 5.052 0 6.97l-16.834 16.9c24.204 25.939 39.396 60.424 40.61 98.477h23.816zM359.25 247.77c61.624 0 111.76 50.343 111.76 112.229 0 1.752-.051 3.49-.135 5.223a5.944 5.944 0 0 1-2.134 4.306 5.848 5.848 0 0 1-4.598 1.321 12.685 12.685 0 0 0-12.572 5.96c-3.67 6-1.8 13.88 4.172 17.566a12.55 12.55 0 0 0 2.823 1.28 5.942 5.942 0 0 1 3.592 3.182 5.991 5.991 0 0 1 .078 4.819c-3.056 7.254-6.95 14.275-11.578 20.877a5.943 5.943 0 0 1-4.085 2.481 5.932 5.932 0 0 1-4.578-1.345c-.48-.399-.979-.789-1.522-1.127a12.605 12.605 0 0 0-9.611-1.54 12.614 12.614 0 0 0-7.878 5.732 12.792 12.792 0 0 0 1.05 14.83 5.964 5.964 0 0 1 1.31 4.615 5.972 5.972 0 0 1-2.498 4.09 112.016 112.016 0 0 1-19.324 10.79 5.904 5.904 0 0 1-4.832-.056 5.948 5.948 0 0 1-3.19-3.64 12.737 12.737 0 0 0-5.46-7.067 12.612 12.612 0 0 0-9.61-1.54 12.631 12.631 0 0 0-7.88 5.732l-.003.003a12.806 12.806 0 0 0-1.713 8.684 6 6 0 0 1-1.247 4.684 5.959 5.959 0 0 1-4.303 2.203c-1.997.101-4.008.164-6.034.164-61.627 0-111.763-50.342-111.763-112.227 0-61.886 50.136-112.228 111.763-112.228zm84.81 165.202c1.785-2.868 3.42-5.81 4.891-8.807-11.089-7.273-14.47-22.193-7.496-33.595a24.464 24.464 0 0 1 4.96-5.792h-28.033c-2.312 29.079-25.397 52.26-54.358 54.58v28.614a24.363 24.363 0 0 1 13.65-8.81c6.385-1.531 12.99-.476 18.581 2.979a24.56 24.56 0 0 1 7.714 7.557 99.744 99.744 0 0 0 7.427-4.156 24.742 24.742 0 0 1 1.475-23.05c3.431-5.615 8.847-9.556 15.233-11.084a24.246 24.246 0 0 1 15.956 1.564zm-184.58-48.194c1.128 24.057 10.731 45.923 25.878 62.647l28.713-28.83a59.431 59.431 0 0 1-13.954-33.808h-37.67c-.012 0-.021-.009-.033-.009h-2.934zm25.887-72.22c-15.15 16.72-24.756 38.59-25.887 62.646h40.635a59.442 59.442 0 0 1 13.938-33.82l-26.624-26.738c-.012-.009-.015-.03-.027-.042l-2.035-2.047zm69.12-29.658v-3.09c-23.95 1.137-45.725 10.774-62.379 25.981l28.692 28.812c9.227-7.887 20.877-12.984 33.681-14.006v-37.664c0-.012.006-.024.006-.033zm43.216 51.712l28.685-28.809c-16.648-15.207-38.414-24.853-62.364-25.992v40.786c12.805 1.025 24.455 6.125 33.679 14.015zm-38.453 90.356c24.74 0 44.803-20.143 44.803-44.99 0-24.848-20.062-44.988-44.803-44.988-24.747 0-44.806 20.14-44.806 44.988 0 24.847 20.059 44.99 44.806 44.99zm-38.435.396l-28.713 28.833c16.651 15.21 38.432 24.847 62.385 25.983v-40.82c-12.798-1.021-24.448-6.112-33.672-13.996zm97.57-50.16h40.62c-1.138-24.05-10.731-45.91-25.873-62.629l-28.688 28.81a59.445 59.445 0 0 1 13.941 33.819zM1108.2 313.19v94.305h74.274v13.329h-87.514V313.19h13.24zm-153.805 0v94.305h74.292v13.329h-87.532V313.19h13.24zm-318.875 0h98.695v13.325h-42.705v94.309h-13.27v-94.309h-42.72V313.19zm786.95 60.882l-.29.003v46.75h-13.273V313.19h67.405v-.045c16.597 0 30.052 13.464 30.052 30.152 0 16.649-13.455 30.166-30.052 30.166h-3.853l35.955 47.361h-17.072l-35.564-47.36-33.308.608zm-.29-47.556v33.82l54.132-.186c9.787 0 17.62-8.23 16.758-18.27-.758-8.829-8.6-15.364-17.424-15.364h-53.465zm-595.372 94.308V313.19h13.276v107.634h-13.276zm421.938-107.634h87.526v13.325h-74.247v33.82h47.289v13.317h-47.289v33.843h74.247v13.329h-87.526V313.19z"
      fill="white"
      fillRule="evenodd"
    />
  </svg>
</Logo>
`;

exports[`<Logo /> should render withouth a problem 1`] = `
<Logo
  color="white"
  height="59"
  theme={
    Object {
      "breakpoints": Object {
        "lg": 992,
        "md": 768,
        "sm": 576,
        "xl": 1200,
        "xs": 0,
      },
      "dimensions": Object {
        "big": "3.1rem",
        "bigInt": 3.1,
        "huge": "4.5rem",
        "hugeInt": 4.5,
        "medium": "1.5rem",
        "mediumInt": 1.5,
        "radius": "0.4rem",
        "radiusInt": 0.4,
        "small": "0.7rem",
        "smallInt": 0.7,
        "tiny": "0.3rem",
        "tinyInt": 0.3,
      },
      "fonts": Object {
        "family": "\\"Open Sans\\", Helvetica, Arial, sans-serif",
        "size": Object {
          "big": "1.6rem",
          "bigInt": 1.6,
          "default": "1.2rem",
          "defaultInt": 1.2,
          "h1": "3.6rem",
          "h1Int": 3.6,
          "h2": "3rem",
          "h2Int": 3,
          "h3": "2.6rem",
          "h3Int": 2.6,
          "h4": "2.2rem",
          "h4Int": 2.2,
          "h5": "2.0rem",
          "h5Int": 2,
          "h6": "1.8rem",
          "h6Int": 1.8,
          "huge": "6.4rem",
          "hugeInt": 6.4,
          "medium": "1.4rem",
          "mediumInt": 1.4,
          "small": "0.7rem",
          "smallInt": 0.7,
          "tiny": "0.3rem",
          "tinyInt": 0.3,
        },
        "spacing": Object {
          "big": "1.5px",
          "medium": "1px",
          "small": "0.5px",
        },
        "weight": Object {
          "normal": 400,
          "thick": 600,
          "thin": 200,
        },
      },
      "palette": Object {
        "avatar": Object {
          "alizarin": "hsl(6, 78%, 57%)",
          "carrot": "hsl(28, 80%, 52%)",
          "emerald": "hsl(145, 63%, 49%)",
          "river": "hsl(204, 70%, 53%)",
          "turquoise": "hsl(168, 76%, 42%)",
          "wisteria": "hsl(282, 44%, 47%)",
        },
        "black": "hsl(0, 0%, 0%)",
        "bodyBackground": "hsl(218, 29%, 95%)",
        "darkBlue": "hsl(213, 17%, 20%)",
        "darkGrey": "hsl(206,16%,60%)",
        "failure": Object {
          "dark": "#e74432",
          "darker": "#e42f1b",
          "default": "hsl(6, 79%, 65%)",
        },
        "googleBrandRed": "hsl(5, 81%, 56%)",
        "lightGrey": "hsl(207,22%,90%)",
        "mediumGrey": "hsl(206,23%,69%)",
        "mysticGrey": "hsl(216,45%,98%)",
        "primary": Object {
          "dark": "#1873a0",
          "darker": "#14638a",
          "default": "hsl(200, 74%, 46%)",
        },
        "secondary": Object {
          "default": "hsl(0, 100%, 100%)",
        },
        "spaceGrey": "hsl(207,13%,45%)",
        "success": Object {
          "dark": "#6da333",
          "darker": "#60902d",
          "default": "hsl(89, 53%, 52%)",
        },
        "warning": Object {
          "dark": "#f59a24",
          "darker": "#f48f0b",
          "default": "hsl(34, 91%, 65%)",
        },
        "white": "hsl(0, 100%, 100%)",
        "whiteOpacity": [Function],
      },
    }
  }
  width="180"
>
  <svg
    height="59"
    viewBox="0 0 1689 720"
    width="180"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M359.247 496.87c75.28 0 136.3-61.28 136.3-136.874 0-75.59-61.02-136.87-136.3-136.87-75.276 0-136.3 61.28-136.3 136.87 0 75.594 61.024 136.874 136.3 136.874zm174.341-141.8c2.701 0 4.91 2.217 4.91 4.926 0 2.714-2.209 4.932-4.91 4.932h-23.815c-1.215 38.053-16.407 72.541-40.61 98.473l16.833 16.904c1.91 1.915 1.91 5.055 0 6.973a4.915 4.915 0 0 1-6.944 0l-16.83-16.907c-25.825 24.309-60.171 39.561-98.063 40.787v23.91c0 2.712-2.211 4.932-4.912 4.932-2.7 0-4.912-2.22-4.912-4.932v-23.91c-37.891-1.226-72.235-16.478-98.06-40.787l-16.832 16.907a4.915 4.915 0 0 1-6.944 0c-1.907-1.918-1.907-5.058 0-6.973l16.833-16.904c-24.2-25.932-39.393-60.42-40.61-98.473h-23.813c-2.7 0-4.909-2.218-4.909-4.932 0-2.71 2.208-4.927 4.909-4.927h23.813c1.217-38.053 16.41-72.538 40.61-98.476l-16.833-16.901c-1.907-1.918-1.907-5.055 0-6.97a4.904 4.904 0 0 1 6.944 0l16.833 16.9c25.824-24.305 60.168-39.56 98.06-40.78v-23.916c0-2.709 2.21-4.926 4.911-4.926 2.7 0 4.912 2.217 4.912 4.926v23.916c37.892 1.22 72.238 16.475 98.062 40.78l16.83-16.9a4.904 4.904 0 0 1 6.945 0c1.91 1.915 1.91 5.052 0 6.97l-16.834 16.9c24.204 25.939 39.396 60.424 40.61 98.477h23.816zM359.25 247.77c61.624 0 111.76 50.343 111.76 112.229 0 1.752-.051 3.49-.135 5.223a5.944 5.944 0 0 1-2.134 4.306 5.848 5.848 0 0 1-4.598 1.321 12.685 12.685 0 0 0-12.572 5.96c-3.67 6-1.8 13.88 4.172 17.566a12.55 12.55 0 0 0 2.823 1.28 5.942 5.942 0 0 1 3.592 3.182 5.991 5.991 0 0 1 .078 4.819c-3.056 7.254-6.95 14.275-11.578 20.877a5.943 5.943 0 0 1-4.085 2.481 5.932 5.932 0 0 1-4.578-1.345c-.48-.399-.979-.789-1.522-1.127a12.605 12.605 0 0 0-9.611-1.54 12.614 12.614 0 0 0-7.878 5.732 12.792 12.792 0 0 0 1.05 14.83 5.964 5.964 0 0 1 1.31 4.615 5.972 5.972 0 0 1-2.498 4.09 112.016 112.016 0 0 1-19.324 10.79 5.904 5.904 0 0 1-4.832-.056 5.948 5.948 0 0 1-3.19-3.64 12.737 12.737 0 0 0-5.46-7.067 12.612 12.612 0 0 0-9.61-1.54 12.631 12.631 0 0 0-7.88 5.732l-.003.003a12.806 12.806 0 0 0-1.713 8.684 6 6 0 0 1-1.247 4.684 5.959 5.959 0 0 1-4.303 2.203c-1.997.101-4.008.164-6.034.164-61.627 0-111.763-50.342-111.763-112.227 0-61.886 50.136-112.228 111.763-112.228zm84.81 165.202c1.785-2.868 3.42-5.81 4.891-8.807-11.089-7.273-14.47-22.193-7.496-33.595a24.464 24.464 0 0 1 4.96-5.792h-28.033c-2.312 29.079-25.397 52.26-54.358 54.58v28.614a24.363 24.363 0 0 1 13.65-8.81c6.385-1.531 12.99-.476 18.581 2.979a24.56 24.56 0 0 1 7.714 7.557 99.744 99.744 0 0 0 7.427-4.156 24.742 24.742 0 0 1 1.475-23.05c3.431-5.615 8.847-9.556 15.233-11.084a24.246 24.246 0 0 1 15.956 1.564zm-184.58-48.194c1.128 24.057 10.731 45.923 25.878 62.647l28.713-28.83a59.431 59.431 0 0 1-13.954-33.808h-37.67c-.012 0-.021-.009-.033-.009h-2.934zm25.887-72.22c-15.15 16.72-24.756 38.59-25.887 62.646h40.635a59.442 59.442 0 0 1 13.938-33.82l-26.624-26.738c-.012-.009-.015-.03-.027-.042l-2.035-2.047zm69.12-29.658v-3.09c-23.95 1.137-45.725 10.774-62.379 25.981l28.692 28.812c9.227-7.887 20.877-12.984 33.681-14.006v-37.664c0-.012.006-.024.006-.033zm43.216 51.712l28.685-28.809c-16.648-15.207-38.414-24.853-62.364-25.992v40.786c12.805 1.025 24.455 6.125 33.679 14.015zm-38.453 90.356c24.74 0 44.803-20.143 44.803-44.99 0-24.848-20.062-44.988-44.803-44.988-24.747 0-44.806 20.14-44.806 44.988 0 24.847 20.059 44.99 44.806 44.99zm-38.435.396l-28.713 28.833c16.651 15.21 38.432 24.847 62.385 25.983v-40.82c-12.798-1.021-24.448-6.112-33.672-13.996zm97.57-50.16h40.62c-1.138-24.05-10.731-45.91-25.873-62.629l-28.688 28.81a59.445 59.445 0 0 1 13.941 33.819zM1108.2 313.19v94.305h74.274v13.329h-87.514V313.19h13.24zm-153.805 0v94.305h74.292v13.329h-87.532V313.19h13.24zm-318.875 0h98.695v13.325h-42.705v94.309h-13.27v-94.309h-42.72V313.19zm786.95 60.882l-.29.003v46.75h-13.273V313.19h67.405v-.045c16.597 0 30.052 13.464 30.052 30.152 0 16.649-13.455 30.166-30.052 30.166h-3.853l35.955 47.361h-17.072l-35.564-47.36-33.308.608zm-.29-47.556v33.82l54.132-.186c9.787 0 17.62-8.23 16.758-18.27-.758-8.829-8.6-15.364-17.424-15.364h-53.465zm-595.372 94.308V313.19h13.276v107.634h-13.276zm421.938-107.634h87.526v13.325h-74.247v33.82h47.289v13.317h-47.289v33.843h74.247v13.329h-87.526V313.19z"
      fill="white"
      fillRule="evenodd"
    />
  </svg>
</Logo>
`;
